openstack/nova/templates/hypervisors/hypervisors-vmware-deployment-vct.yaml

```diff
@@ -71,39 +71,9 @@ template: |
           {{- end }}
           volumeMounts:
           - mountPath: /etc/nova
-            name: etcnova
-          - mountPath: /etc/nova/nova.conf
             name: nova-etc
-            subPath: nova.conf
-            readOnly: true
-          - mountPath: /etc/nova/policy.json
-            name: nova-etc
-            subPath: policy.json
-            readOnly: true
-          - mountPath: /etc/nova/logging.ini
-            name: nova-etc
-            subPath: logging.ini
-            readOnly: true
-          - mountPath: /etc/nova/nova-compute.conf
-            name: hypervisor-config
-            subPath: nova-compute.conf
-            readOnly: true
-          - mountPath: /etc/nova/hostgroup-reservations.json
-            name: hypervisor-config
-            subPath: hostgroup-reservations.json
-            readOnly: true
-          - mountPath: /etc/sudoers
-            name: nova-etc
-            subPath: sudoers
-            readOnly: true
-          - mountPath: /etc/nova/rootwrap.conf
-            name: nova-etc
-            subPath: rootwrap.conf
-            readOnly: true
-          - mountPath: /etc/nova/rootwrap.d/compute.filters
-            name: nova-etc
-            subPath: compute.filters
-            readOnly: true
+          - mountPath: /etc
+            name: sudoers
```

Comment:
Doing this, we mount one configmap over another again, as we mount this to `/etc` and then `nova-etc` to `/etc/nova`. We don't want that. Can we find another solution? Maybe mount the whole `/etc` because of `sudoers` and prepend all paths for Nova's config files with `nova/`?

Reply:
But then, whole /etc on the container will get overwritten with what's on our projected volume.

Reply:
The way would be to use subPath for /etc/sudoers.